### PR TITLE
Completely changed subject and added some ASCII art

### PR DESCRIPTION
**Pull request by floridop**
Added some content to your story. You are free to ask me to do any change you like _before_ merging! Use this opportunity to experiment.

To complete this task you need to merge my contribution as explained in slides 95-101 keeping in mind that you are the release manager.

### CODE REVIEW

File: fairytale.md [hunk 1]
```diff
@@ -1,4 +1,45 @@
 # MNXB11 fairytale
 
 Once upon a time,
-there was a **beautiful** _princess_
```
Comment (Takumi0104) [inline fairytale.md:4]: There was a man who played piano very well.

**Comment by Takumi0104** [comment]
Hi, Floridop. Now, I converted my fairytale.md included ASCII art as a guideline, which is said to be added to ASCII art in my original. 
And I'm going to merge the pull script. 
From the above, does my process have any problem?

**Comment by floridop** [comment]
Hi Takumi, I wish I could answer you but I am having hard time understanding your choice of words.

I never asked you to convert anything so I wonder what do you mean by "converted"

A guideline is a set of instructions to do something, how can an ASCII art be a guideline? 

Trying to interpret what you wrote, I think you mean you did something regarding my suggestion for changes in this pull request. I unfortunately don't understand what is it that you did from your words 😔

Then you said you're going to "merge the pull script". But there is no script. Maybe you meant "merge the pull request"?

As an outcome of this homework it is important that you learn the proper words to describe what you are doing with git. We can discuss this in person later today. 

So right now I can only answer that I do not understand your process so I cannot help you here. Let's discuss in person and you show me what you did.

The only process required is the one described in the slides. If there are slides you do not understand please refer to those then it's easier for me to explain.

**Comment by Takumi0104** [comment]
Hi, Floridop. Thank you for pointing out that my message lacked detail.

Here's a more concrete explanation of my question:

I modified the original fairytale.md file by adding ASCII art, and I pushed it to the alttale directory in GitHub. Now, I'm about to merge the pull request, but I'm feeling anxious about the process—especially whether it's correct to have modified the original fairytale.md.

Because of this anxiety, I don't want to merge the pull request until I'm sure everything is correct. Could you please provide guidance on whether it's okay to proceed?

**Comment by floridop** [comment]
Hi @Takumi0104 

I think there is some misunderstanding on how the _git flow_ we are using works. It's better if we meet and I explain you the whole git flow. Do not merge anything right now. Please read again slides 81, 84-86.

The objective of this exercise is for you to look at my modifications (by clicking on the [Files Changed](https://github.com/Takumi0104/MNXB11-homework/pull/1/files) menu on top of this conversation ) and decide if you want to include them in your _main_ . 
This exercise has nothing to do with your _alttale_ repository, it has to do with **your** _main_ and **my** _altstory_ .
You do not need to edit the files directly. You need to look at the differences on GitHub as shown in slides 95-101.


Regards
Florido

**Comment by floridop** [comment]
I'll close this part of the homework, but if there is something else you're
curious to try and I am available I'll be happy to test.

**Pull request merged**